script/plugin_tools.sh

```diff
@@ -2,14 +2,14 @@
 set -ev
 
 if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
-  echo "Running for all packages"
-  pub global run flutter_plugin_tools "$@"
+  echo "Running for all packages ($PLUGIN_SHARDING)"
+  pub global run flutter_plugin_tools "$@" $PLUGIN_SHARDING
 else
   FLUTTER_CHANGED_GLOBAL=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep -v packages | wc -l`
   FLUTTER_CHANGED_PACKAGES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep -o "packages/[^/]*" | sed -e "s/packages\///g" | sort | uniq | paste -s -d, -`
   if [ "${FLUTTER_CHANGED_PACKAGES}" = "" ] || [ $FLUTTER_CHANGED_GLOBAL -gt 0 ]; then
-    echo "Running for all packages"
-    pub global run flutter_plugin_tools "$@"
+    echo "Running for all packages ($PLUGIN_SHARDING)"
+    pub global run flutter_plugin_tools "$@" $PLUGIN_SHARDING
   else
     echo "Running only for $FLUTTER_CHANGED_PACKAGES"
     pub global run flutter_plugin_tools "$@" --plugins=$FLUTTER_CHANGED_PACKAGES
```

Comment:
@mravn-google should `$PLUGIN_SHARDING` be added here as well?

Reply:
That wasn't my original intention, as this branch usually means a small number of plug-ins. Trouble is, the Travis config spawns the same number of jobs either way, so we end up with redundant execution.

Reply:
Got it! I was just wondering if `$PLUGIN_SHARDING` was missed here.